week2/homework/src/index.js

```diff
@@ -1,3 +1,115 @@
 'use strict';
+const fs = require('fs');
+const commandOption = process.argv[2];
+const itemIndex = process.argv[3] - 1;
+const myArguments = process.argv.slice(3);
+const uniCode = 'utf8';
+const myCommandFile = 'command.txt';
```

Comment:
This file is not in your git repository: if I start the program with `node . list` I get an error since this file is not available. 
```
{ Error: ENOENT: no such file or directory, open 'command.txt'
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'command.txt' } 'file not found!'
```
At least add this file to the repository, but even better if you add logic to create the file if it isn't available.